app/assets/stylesheets/components/autocomplete.scss

```diff
@@ -2,6 +2,11 @@
 @import '@reach/combobox/styles';
 
 .crayons-autocomplete {
+  // Override of the default hidden class to make sure it takes priority over any display style attached to the 'replaceElement' textarea
+  .hidden {
```

Comment:
Is this for when the element has both `crayons-autocomplete` and `hidden` as classes?

If so I think you should include the ampersand 

```css
  &.hidden {
```

Reply:
It's for children of the `crayons-autocomplete` class that have the 'hidden' class 🙂

Reply:
TIL: that the ampersand is optional for children 👍 
I'd have expected

```scss
.crayons-autocomplete {
  & .hidden {
  
  }
}
```

who knew it was optional 😁

Reply:
yeah I love the nested styling in scss 😍